defaults/install/the-build/build.circleci.yml

```diff
@@ -13,6 +13,3 @@ drupal:
 
 behat:
   args: "--profile=circleci --suite=default --strict --format=junit --out=/tmp/artifacts --tags=~@skipci"
-
```

Comment:
@becw On Hampshire I actually just updated this line to point to the vendor directory -- how come we wouldn't want this to still run during the Circle CI tests?

Reply:
It would still run -- but `vendor/bin/drupal-check` is now the default value.